core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala

```diff
@@ -82,7 +82,7 @@ object ReassignPartitionsCommand extends Logging {
     }
   }
 
-  def generateAssignment(zkUtils: ZkUtils, opts: ReassignPartitionsCommandOptions) {
+  def generateAssignment(zkUtils: ZkUtils, opts: ReassignPartitionsCommandOptions): Map[TopicAndPartition, Seq[Int]] = {
```

Comment:
Can you please elaborate why you think it's not good to move the println to `main`? In fact, that is exactly what was done in the following PR that adds some tests to the reassign partitions command:

https://github.com/apache/kafka/pull/956/files#diff-15a36057cf9f21154f103a55757a7b59R99